test/helm/gateway-helm/envoy-gateway-gateway-namespace-config-watch.out.yaml

```diff
@@ -1011,3 +1011,5 @@ webhooks:
           operator: In
           values:
             - envoy-gateway-system
```

Comment:
Do we need to handle `envoy-gateway-system` in the gateway namespace mode?

Reply:
I think we should allow that happen.

Reply:
Is `envoy-gateway-system` always in the watched ns list? Even it's not explicitly specified? If yes, then this makes sense.

Reply:
make sense.